HW3/Training-HW3-flex/css/styles.css

```diff
@@ -1,124 +1,128 @@
 .container {
   width: 1366px;
-  height: auto;
-  background-color: rgba(0, 0, 0, 0.852);
-  position: relative;
+  background-image: url('../img/backgroundImage.png');
```

Comment:
The `container` shouldn't have the `background` property

Background img should be stretched without width restrictions according to the design